### PR TITLE
Neo adjuvant therapy tx effect lookup (#155)

### PR DESCRIPTION
**Pull request by CookiIMS**

### CODE REVIEW

File: src/main/java/com/imsweb/algorithms/neoadjuvant/NeoAdjuvantTherapyTxEffectUtils.java [hunk 1]
```diff
@@ -107,7 +115,7 @@ private NeoAdjuvantTherapyTxEffectUtils() {
      */
     public static Map<String, String> getLookup(String schemaId) {
         if (StringUtils.isBlank(schemaId))
-            return null;
+            return Collections.emptyMap();
```
Comment (depryf) [inline src/main/java/com/imsweb/algorithms/neoadjuvant/NeoAdjuvantTherapyTxEffectUtils.java:118]: @CookiIMS why did you change the returned value in this case? It's a change in behavior that could easily introduced a bug in whatever software calls it.

Comment (CookiIMS) [reply src/main/java/com/imsweb/algorithms/neoadjuvant/NeoAdjuvantTherapyTxEffectUtils.java:118]: It was a SonarQube recommendation, it didn't break any tests so I didn't think twice about it. I'll change it back.

Comment (CookiIMS) [reply src/main/java/com/imsweb/algorithms/neoadjuvant/NeoAdjuvantTherapyTxEffectUtils.java:118]: Actually now that I think about it it did break some tests and I changed them. I think I just put too much trust in SonarQube here lol.

Comment (CookiIMS) [reply src/main/java/com/imsweb/algorithms/neoadjuvant/NeoAdjuvantTherapyTxEffectUtils.java]: Should be fixed now.
